Responsive Clone of The Nextweb Homepage

t-yanick opened the pull request:
- We made extensive use of grids and flexbox
- We applied @media queries, making use of two breakpoints (tablet view and mobile view)

Commit:
Added index.html and style.html files

t-yanick committed:
created nav bars and first 2 sections in HTML page layout

Commit:
Added Latest posts ad Funding Sections

Commit:
Added images and li classes in latest and funding sections

t-yanick committed:
added latest post section and content

Commit:
Added Deals and Footer Sections, Images and Content

t-yanick committed:
started styling navbar-1 and navbar-2

Commit:
added grid and  css on the header-articles and latest articles

Commit:
added css on deal section

t-yanick committed:
started styling funding section

Commit:
added paddings and margins on the 2 sections

Commit:
fixed bottom text on the popular section

t-yanick committed:
styled popular section and updated html in latest section

Commit:
Added css to latest-article section

t-yanick committed:
completed styling of latest deals section and started footer

Commit:
fixed bottom footer and CSS

Commit:
fixed responsive nav bar

t-yanick committed:
fixed text, gradient issues in main articles section

Commit:
fixed latest news section and CSS

t-yanick committed:
added styling to latest funding section

Commit:
fixed some css errors on funding section

t-yanick committed:
fixed funding section width

t-yanick committed:
added media query for nav bars

Commit:
fixed tablet media query for the Latest News section

t-yanick committed:
added media query breakpoints for funding and popular sections

Commit:
added CSS to latest news article mobile reposnsive

Commit:
Fixed responsivness

Commit:
Fixed responsivness

Commit:
Added Responsive BG images and Media Query

t-yanick committed:
fixed footer

Commit:
Added Content in the Readme file

t-yanick committed:
added linters

t-yanick committed:
fixed linters error

t-yanick committed:
fixed more linters error

t-yanick committed:
fixed more linters error

t-yanick committed:
fixed more linters error

t-yanick committed:
fixed more linters error

t-yanick committed:
fixed more linters error

kitpao commented on style.css:
```diff
@@ -0,0 +1,1682 @@
+  line-height: 21px;
+  position: absolute;
+  top: 50px;
+}
```
- [ ] Please adjust your width, the light orange is overflowing.

kitpao reviewed (changes requested):
# 👋 Hello @t-yanick and @bmuhamood 

This is your feedback for this pull request, most of them are about missing objects, positioning and sizing, check them below:

**_Note:_** All the **[OPTIONAL]** are **not required** in order to approve your project but are personal suggestions to improve your coding experience

- [ ] For next time, remember to write your commits in the imperative tense, as stated in [the article pointed in pathwright](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

- [ ] Your linters are displaying errors, please correct them until you get a message similar to this one:
![image](https://user-images.githubusercontent.com/53256139/88387134-78389d80-cd77-11ea-91c9-d1deb84ef926.png)

- [x] Please add .DS_Store to your gitignore file
![image](https://user-images.githubusercontent.com/53256139/89334699-88e0f180-d65c-11ea-9ac8-342bb82cf0b3.png)

- [ ] **Optional** For good HTML semantics, try to use `<nav>`, `<main>`, and `<footer>` tags

### Mobile screens:
- [x] Please check the following differences and fix them
![image](https://user-images.githubusercontent.com/53256139/89330257-9ba3f800-d655-11ea-8f9a-8a29b6bff943.png)
![image](https://user-images.githubusercontent.com/53256139/89332308-d0fe1500-d658-11ea-877d-af8666f1ca74.png)
![image](https://user-images.githubusercontent.com/53256139/89332655-613c5a00-d659-11ea-8ff9-093ef32e2891.png)
![image](https://user-images.githubusercontent.com/53256139/89332902-c85a0e80-d659-11ea-8953-0320133da045.png)

### Tablet screens
- [x] Please fix the following:
![image](https://user-images.githubusercontent.com/53256139/89333655-e3794e00-d65a-11ea-8d29-12e92f1ff6a8.png)
![image](https://user-images.githubusercontent.com/53256139/89333871-3ce17d00-d65b-11ea-804d-26766d4abdc3.png)

### Desktop screens
- [x] Please fix the following differences
![image](https://user-images.githubusercontent.com/53256139/89334302-ec1e5400-d65b-11ea-92d2-320ac179c076.png)
__

- [ ] Your CSS looks familiar to me, did you just copy-pasted from this repo? https://github.com/digitekh2i/Building-with-Responsive-Design/blob/develop/css/main.css
Remember [Plagiarism Guidelines of Microverse](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/85215970/)

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

Commit:
added logo image

t-yanick committed:
fixed elements positioning in the tablet and desktop views

Commit:
worked on positioning issues on mobile and Tablet views

t-yanick committed:
fixed CSS indentation

Commit:
added fixed logo on mobile view while scrolling

t-yanick committed:
fixed image widths and added correct logo

Commit:
fixed HTML linter errors

Commit:
fixed linter errors and also removed lines of codes from other sources

Commit:
Fixed more linter errors and image gradients

Commit:
fixed linters errors in CSS

Commit:
Fixed more linter errors

t-yanick committed:
fixed lines and indentation issues

t-yanick committed:
fixed linter background value error and empty lines

t-yanick committed:
fixed linter background value error and empty lines

t-yanick commented:
This particular linter error cannot be fixed. we have tried to no avail

 836:1  ✖  Unexpected missing end-of-source newline   no-missing-end-of-source-newline

expjazz commented on index.html:
```diff
@@ -0,0 +1,715 @@
+                        <div class="tmw-logo"><img src="./images/tnw.svg" alt="logo"></div>
+                    </div>
+                    <p class="current-date">Wednesday - 5, August 2020</p>
+                    <div class="headings">INSIGHTS</div>
+                    <div class="head-text-title">Kodak, now a pharma company, expects drugs to make up for 30-40% of its
+                        future business.</div>
+                    <div class="head-text-bottom">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago <i
+                            class="fas fa-share-alt"></i> 6</div>
+                </div>
+            </div>
+        </div>
```
- [ ] Now that you made the navbar fix, you have a problem with this section (it is under the navbar). Also re check the font size on the screen sizes in between the breakpoints. 

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89460579-e0588d80-d740-11ea-99b3-b25ed3772151.png) |![image](https://user-images.githubusercontent.com/61479648/89460671-0a11b480-d741-11ea-8309-2f688d1ca37a.png)  |

#### also in Mobile

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89460859-5e1c9900-d741-11ea-8853-f0c93bd62120.png) |![image](https://user-images.githubusercontent.com/61479648/89460877-6b398800-d741-11ea-9544-9b2394633f73.png)  |

expjazz commented on index.html:
```diff
@@ -0,0 +1,715 @@
+            </div>
+        </div>
+    </div>
+    <footer>
+        <div class="footer-1">
+            <div class="icons">
+                <i class="fab fa-facebook-square"></i>
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-youtube"></i>
+                <i class="fab fa-instagram"></i>
+                <i class="fab fa-pinterest-p"></i>
+                <i class="fas fa-wifi"></i>
+                <i class="fas fa-envelope"></i>
+            </div>
+            <div class="footer-list">
```
- [ ] After the last change, the layout is breaking in the mobile screen size.


| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/89461080-b94e8b80-d741-11ea-966c-7fbb69d2bf1a.png)| ![image](https://user-images.githubusercontent.com/61479648/89461128-cc615b80-d741-11ea-9945-dce0faf6565e.png) |

expjazz reviewed (changes requested):
## Status: Changes Required :recycle: 

Hey team! You did a very good job on this project, and on the changes required by my colleague. The page looks really great.

About the linter, I fixed here on my machine. Install the locally as is in here:
![image](https://user-images.githubusercontent.com/61479648/89461452-632e1800-d742-11ea-954a-bdfaa915e576.png)

Then run _npx stylelint "**/*.{css,scss}" --fix_ 

If the problem persists, feel free to reach on slack. I also pointed out a few changes in the inline comments below. Kindly fix them also.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

t-yanick committed:
fixed footer, red logo and font size in first section

t-yanick committed:
fixed some litter issues

t-yanick committed:
fixed more linter issues

t-yanick commented:
The linter errors we have now are probably errors related to configurations

elmejdki reviewed (changes requested):
# status: require changes :recycle: 
## Great job so far team :clap: :+1: :100:, you are almost there, you need to make just some minor changes :hammer: :wrench: :toolbox: to make your project approved :checkered_flag: 

- [ ] the style lint error is not related to configuration please fix it, here are the errors:
![image](https://user-images.githubusercontent.com/34653764/89474616-04868f80-d77e-11ea-9590-5f07ecf7a7ce.png)

- [x] you have a horizontal scrolling on the tablet view please fix it,
![image](https://user-images.githubusercontent.com/34653764/89475324-ba061280-d77f-11ea-9470-a209d4c2be63.png)

- [x] also on the mobile view there is horizontal scrolling,
![image](https://user-images.githubusercontent.com/34653764/89475345-c7bb9800-d77f-11ea-9998-4c25f9c4144c.png)

- [x] Fix the hero section and the TNW logo on the navbar, they should look like the original:

| *Clone* | *Original* |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/34653764/89475511-30a31000-d780-11ea-95b9-f3d4533a72b0.png) | ![image](https://user-images.githubusercontent.com/34653764/89475482-19fcb900-d780-11ea-8a3c-62ccf93928f4.png) |

- [x] also on tablet view they are not showing correctly:

| *Clone* | *Original* |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/34653764/89475603-6647f900-d780-11ea-872b-0b436953ecb5.png) | ![image](https://user-images.githubusercontent.com/34653764/89475631-719b2480-d780-11ea-8a72-92610a57d433.png) |

- [x] Fix this title on tablet view:

| *Clone* | *Original* |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/34653764/89475837-fd14b580-d780-11ea-95e0-97b2eb5a8522.png) | ![image](https://user-images.githubusercontent.com/34653764/89475860-0d2c9500-d781-11ea-9eea-f4e5b5a0346f.png) |

- [x] Fix the image sizes on the mobile view for the latest news and latest deals sections.

- [x] I have also noticed that you didn't make the required changes by the previous TSE if you don't make the required changes your project will not be approved.

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

t-yanick committed:
fixed overflow issues and nav tabs in latest deal section

t-yanick committed:
fixed linter errors

t-yanick committed:
fixed image sizes on the mobile view for the latest news and latest deals sections

t-yanick commented:
Fixed requested but seems my terminal is having issues running npm linters for lighthouse, weblint and stylelint
![image](https://user-images.githubusercontent.com/62849872/89527062-19cfde00-d7bf-11ea-98a2-8fa63cdccd97.png)

t-yanick committed:
fixed linter stylelint errors

t-yanick committed:
fixed linter stylelint errors

t-yanick committed:
fixed linter stylelint errors

t-yanick commented:
Looking at our CSS and looking at the screenshot of errors below, we don't really know how we correct these errors for we have tried all options. Thanks for your help on this.

![image](https://user-images.githubusercontent.com/62849872/89529262-ccee0680-d7c2-11ea-87cb-b45f6ff9de99.png)

t-yanick commented:
Our CSS for the section giving linter errors is as below. We cannot figure out what is wrong with styles in the code.

![image](https://user-images.githubusercontent.com/62849872/89529396-09b9fd80-d7c3-11ea-8add-775bc8a52e3c.png)

bertil291utn commented on style.css:
```diff
@@ -0,0 +1,989 @@
+#hidden-2 {
+  display: none;
+}
+
+@media only screen and (max-width: 1024px) {
+  .latest-news-sec {
+    grid-template-columns: repeat(3, 1fr);
+  }
```
- [ ] Your breakpoint for large screen starts on 1025px, should be 1024px. Please change to `@media only screen and (max-width: 1023px) {}` to achieve this

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| ![image](https://user-images.githubusercontent.com/24902525/89531830-77136180-d7b6-11ea-9609-e6bb0582f516.png) | ![image](https://user-images.githubusercontent.com/24902525/89531811-6b279f80-d7b6-11ea-8894-e657ce2a73c3.png) |

bertil291utn commented on style.css:
```diff
@@ -0,0 +1,989 @@
+
+nav {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  display: flex;
+  background-color: black;
+  justify-content: space-between;
+  z-index: 10;
+}
+
+nav ul,
+footer ul {
+  display: flex;
+  list-style-type: none;
```
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display: flex;
}
```
**Html file**

```
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

bertil291utn commented on style.css:
```diff
@@ -0,0 +1,989 @@
+  font-size: 14px;
+}
+
+.f-title li {
+  color: #f42;
+  padding: 0 10px;
+}
+
+.group-list li {
+  color: #f42;
+  font-size: 12px;
+  border-bottom: 1px solid #eee;
```
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: #f42;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-x{
  color: #f42;
}
```
**Html file**

```
<div class="color-x">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

bertil291utn commented on style.css:
```diff
@@ -0,0 +1,989 @@
+
+.news-source {
+  display: flex;
+  font-size: 13px;
+  color: rgb(163, 162, 162);
+}
+
+.deals-source {
+  display: flex;
+}
+
+.price {
+  font-size: 13px;
+  color: rgb(163, 162, 162);
+}
```
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: rgb(163, 162, 162);` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-y{
  color: rgb(163, 162, 162);
}
```
**Html file**

```
<div class="color-y">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

bertil291utn commented on style.css:
```diff
@@ -0,0 +1,989 @@
+  height: 9px;
+}
+
+.icons > * {
+  color: white;
+  background-color: #333;
+  border-radius: 50%;
+  padding: 12px;
+  border: 1px solid #333;
+  margin: 0 8px;
+}
+
+.footer-1 > * {
+  padding: 15px 0;
+}
+
+.after-image > * {
+  color: #f42;
+  font-size: 14px;
+}
+
+.group-title-stuff > * {
+  color: white;
+  padding-right: 14px;
+}
```
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: white;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.white-color{
  color: white;
}
```
**Html file**

```
<div class="white-color">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

bertil291utn commented on style.css:
```diff
@@ -0,0 +1,989 @@
+}
+
+.funding-info {
+  margin: 20px 0;
+  display: grid;
+  width: 100%;
+  grid-template-columns: repeat(4, 1fr);
+  grid-column-gap: 25px;
```
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:grid;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.d-grid{
  display: grid;
}
```
**Html file**

```
<div class="d-grid">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

bertil291utn reviewed (changes requested):
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

### General requeriments
- [ ] Remove `.DS_STORE` directory from your project, please do it first. We don't want a code review just for that requirement.
- [ ] Fix the linter errors
- [ ] Edit your pull request message, follow this [example](https://cdn.filestackcontent.com/osvFJEkQHWQsgDe0PJoQ)

### Specific requeriments

| Section/Screen                             | Large(1024px) | Medium(768px) | Small |
| :----------------------------------------- | :---: | :----: | :---: |
| Navigation bar                             |       |    4    |    6   |
| Top 3 articles section                     | 1     |        |  7     |
| Latest news section                        |       |        |       |
| Latest Funding section                     |   2    |   5     |       |
| Categories (Apps, Gear, Tech, etc) section |       |        |       |
| Latest deals section                       |       |        |       |
| Footer                                     |       |        |   8    |
###### This table is given each section with his screen width approach, each issue is given by number below this table

1. Should have more height to see the logo

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| ![image](https://user-images.githubusercontent.com/24902525/89532119-e5f0ba80-d7b6-11ea-8cb7-c12bfa9966ed.png) | ![image](https://user-images.githubusercontent.com/24902525/89532096-dec9ac80-d7b6-11ea-972d-3d21a0ac2a12.png) |

2. Links section should be below the title
![image](https://user-images.githubusercontent.com/24902525/89532298-2f410a00-d7b7-11ea-80ac-43a83e4487dc.png)

3. Remove extra padding you have in the entire body with a medium screens approach.
![image](https://user-images.githubusercontent.com/24902525/89532477-75966900-d7b7-11ea-86b6-c86d5a435eb2.png)

4. About links shouldn't be in there

![image](https://user-images.githubusercontent.com/24902525/89532557-965ebe80-d7b7-11ea-8f54-1b6602a75a32.png)

5. Remove extra text in this section 
![image](https://user-images.githubusercontent.com/24902525/89532608-aeced900-d7b7-11ea-9abb-7a99c5d490bb.png)
 
6. Fix the logo, TNW text should be in the center, also hegith is too much please change them

![image](https://user-images.githubusercontent.com/24902525/89532820-01a89080-d7b8-11ea-8ab2-9482548a1d8a.png)

7. The Height in each article should be less

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| ![image](https://user-images.githubusercontent.com/24902525/89532880-1f75f580-d7b8-11ea-876c-0710ffc3bb1a.png) | ![image](https://user-images.githubusercontent.com/24902525/89532863-18e77e00-d7b8-11ea-8971-ac73af5228b8.png) |

8. The footer should have less width with this screening approach 

![image](https://user-images.githubusercontent.com/24902525/89533012-564c0b80-d7b8-11ea-9f18-7624d67c43d3.png)

### Summary

-  Remove extra directory `.DS_STORE` you have in your project
- Change media query for large screen should start on 1024px, check inline [comment](https://github.com/t-yanick/building-with-responsive-design/pull/1#discussion_r466376251)
-  Don't forget to work with inline comments below this comment box.

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

t-yanick committed:
added one class for all flex itms and color f42

t-yanick committed:
removed left order on latest deal block links

t-yanick committed:
deleted .DS_Store

t-yanick committed:
added .stylelintrc.json file in the root directory

t-yanick committed:
fixed linter issues

t-yanick commented:
#1 Collaborative Project 6: Building with responsive design.

- Modified 2 breakpoints (max-width for tablet 1023px and 767px for mobile) for tablet and mobile device views.

- Removed repetitive use of flex, f42 color, white color, and other properties.

- Fixed the linter issues and removed .DS_Store

- About links in the black nav bar display in Tablet view.

- Footer links all display in all views inline.

- Links section below funding title and removed borders

t-yanick committed:
fixed logo position

expjazz commented on index.html:
```diff
@@ -0,0 +1,720 @@
+</head>
+
+<body>
+    <nav class="flex">
+        <ul class="flex">
+            <li class="color-y">News</li>
+            <li class="color-y">Events </li>
+            <li class="color-y">Business </li>
+            <li class="color-y">AMAs</li>
+            <li class="color-y">Spaces</li>
+            <li class="color-y">Terms & Conditions</li>
+        </ul>
+        <ul class="flex top-right">
+            <li class="color-y">About</li>
+            <li class="color-y">Advertise</li>
+            <li class="color-y">Jobs</li>
+            <li class="color-y">Contact</li>
+        </ul>
+    </nav>
```
- [ ] Kindly recheck this. You have bugs in many screen sizes. I think the problem lies in the containers, and many of the content seems to be overflowing and not behaving the way they should.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/89556710-2d436f00-d7e8-11ea-9f94-cee779e34833.png)|![image](https://user-images.githubusercontent.com/61479648/89556774-43512f80-d7e8-11ea-95e0-49f4ebff3b9a.png)  |

- [ ] the black border should have full width

![image](https://user-images.githubusercontent.com/61479648/89556897-6976cf80-d7e8-11ea-9226-2d8f12d56e3d.png)

expjazz commented on index.html:
```diff
@@ -0,0 +1,720 @@
+                    <p class="current-date white-color">Wednesday - 5, August 2020</p>
+                    <div class="move-content">
+                        <div class="headings">INSIGHTS</div>
+                        <div class="head-text-title">Kodak, now a pharma company, expects drugs to make up for 30-40% of
+                            its
+                            future business.</div>
+                        <div class="head-text-bottom">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago <i
+                                class="fas fa-share-alt"></i> 6</div>
+                    </div>
+                </div>
+            </div>
+        </div>
```
- [ ] The logo should appear clearly in all pixels bigger than thee tablet view.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89557266-edc95280-d7e8-11ea-8c45-bb959216cbcd.png) | ![image](https://user-images.githubusercontent.com/61479648/89557297-f588f700-d7e8-11ea-8917-95f8f1bee285.png) |

- [ ] Also fix the gradient.

![image](https://user-images.githubusercontent.com/61479648/89557360-10f40200-d7e9-11ea-832b-3aaa7d36e7a0.png)

expjazz commented on index.html:
```diff
@@ -0,0 +1,720 @@
+                </div>
+            </div>
+            <div class="top-grid-3">
+                <div class="child-sec">
+                    <div class="head-text">
+                        <div class="headings">APPS</div>
+                        <div class="headlines">This spooky Deepfake AI mimics dozens of celebs.</div>
+                        <div class="head-text-bottom">by Napier Lopez <i class="far fa-clock"></i> 7 houra ago <i
+                                class="fas fa-share-alt"></i>
+                            11</div>
+                    </div>
+                </div>
```
- [ ] this content needs to be centralized (in the mobile screen size)

![image](https://user-images.githubusercontent.com/61479648/89557982-02f2b100-d7ea-11ea-831d-114e238bd52c.png)

expjazz commented on index.html:
```diff
@@ -0,0 +1,720 @@
+            </div>
+        </div>
+    </div>
+    <footer>
+        <div class="footer-1">
+            <div class="flex icons white-color">
+                <i class="fab fa-facebook-square"></i>
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-youtube"></i>
+                <i class="fab fa-instagram"></i>
+                <i class="fab fa-pinterest-p"></i>
+                <i class="fas fa-wifi"></i>
+                <i class="fas fa-envelope"></i>
+            </div>
+            <div class="flex footer-list">
+                <ul class="flex">
+                    <li class="color-y">EVENTS</li>
```
- [ ] the layout is breaking in the mobile screen size.  (the social media icons should occupy less width than they occupy now).

![image](https://user-images.githubusercontent.com/61479648/89558135-39c8c700-d7ea-11ea-9d2d-31168b7888d2.png)

expjazz reviewed (changes requested):
## Status: Changes Required :recycle: 

Hello again!

You did a very good job on this project, and we are almost done. I think some bugs were created from the last change (you can check them in the inline comments below).

I can see that you worked hard on this project, and it really looks great :1st_place_medal: 

#### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

Commit:
fixed all gradient and all the margin, borders and all reported issues issues in projects as mentioned in the code revies

Commit:
fixed linters issues

bmuhamood commented:
- Fixed all of the above requested changes.

expjazz reviewed (approved):
## Status: Project Approved :clap: 

Congratulations! You did a great job on all the changes requested before. :100: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

bmuhamood commented:
Appreciated a lot. It's been long struggle and appreciated all for your advise.

We have learned a lot.
